es-models/gene_expression/gene_expression.mapping.yaml

```diff
@@ -0,0 +1,31 @@
+_source:
+  excludes:
+  - genes.*
+properties:
+  age_at_diagnosis:
+    type: long
+  case_id:
+    type: keyword
+  days_to_death:
+    type: long
+  ethnicity:
+    type: keyword
+  gender:
+    type: keyword
+  genes:
+    properties:
+      expression_value:
+        type: float
+      gene_id:
+        type: keyword
+    type: nested
+  hgnc_symbol:
+    type: keyword
```

Comment:
The `hgnc_symbol` should be a sibling of `gene_id` in the nested gene doc.

Reply:
oh lol. good catch